Detections/SigninLogs/ExplicitMFADeny.yaml

```diff
@@ -0,0 +1,24 @@
+id: a22740ec-fc1e-4c91-8de6-c29c6450ad00
+name: Explicit MFA Deny
+description: |
+  'User explicitly denies MFA push, indicating that login was not expected and the account's password may be compromised.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AzureActiveDirectory
+    dataTypes:
+      - SigninLogs
+queryFrequency: 1h

```

Comment:
We prefer default to be 1d and allow for users to change as they see fit in the UX.  Not required, but suggested.